CMakeLists.txt

```diff
@@ -670,6 +670,10 @@ include(CPack)
 ### include modules
 
 if (NOT SKIP_TESTS)
+    enable_testing()
+    set(BUILD_TESTING ON)
+    set(CTEST_NEW_FORMAT true)
```

Comment:
what is CTEST_NEW_FORMAT?

Reply:
it affects the name of the ctest configuraiton file it will be `CTestConfiguration.ini` instead of `DartConfiguration.tcl` (lagacy name that isnt clear about what it is for)